src/pages/Home.jsx

```diff
@@ -0,0 +1,90 @@
+import {
+    accessoriesBanner,
+    apparelBanner,
+    equipmentBanner,
+    jerseysBanner,
+    mainBanner,
+    shoesBanner,
+    trophiesBanner,
+} from "../assets/home";
+import { Link } from "react-router-dom";
+import "../styles/Home.css";
+import { TopNavigation } from "../components/TopNavigation.jsx";
+import { CategoryCard } from "../components/CategoryCard.jsx";
+
+export function Home() {
+    const productCategories = [
```

Comment:
Looks good to me. Just add the following objects & import the images at category.js when you integrate mockbee in your project. Category.js would be generated by mockbee itself

Reply:
Sure, did that now